src/components/UI/PageMetadata.tsx

```diff
@@ -0,0 +1,49 @@
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+
+import { SITE_NAME, SITE_URL } from '../../constants';
+
+interface Props {
+  title: string;
+  description: string;
+  image?: string;
+}
+
+export const PageMetadata: React.FC<Props> = ({ title, description, image }) => {
+  const router = useRouter();
+  const url = `${SITE_URL}${router.asPath}`;
+  const fullTitle = `${title} | ${SITE_NAME}`;
```

Comment:
How to handle if no `title` and/or `description` is passed? Should we have a default, or hide those properties?

Reply:
They're currently both required values per the interface above. This is the same pattern we use on the Blog, but on E.org we do make the description optional, and provide a fallback. In both of those cases we require a `title` which I think is appropriate.

If we'd prefer, we could make `description` optional and provide a fallback, but might suggest we leave it as required so the description it most likely to make sense for each page.